org.jitsi.jitsi-meet.metainfo.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id type="desktop">org.jitsi.jitsi-meet</id>
+    <launchable type="desktop-id">org.jitsi.jitsi-meet.desktop</launchable>
+    <name>Jitsi Meet</name>
+    <summary>Jitsi Meet - Secure, Simple and Scalable Video Conferences</summary>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>Apache-2.0</project_license>
+    <developer_name>Jitsi Team</developer_name>
+    <url type="homepage">https://jitsi.org</url>
+    <description>
+        <p>Desktop application for Jitsi Meet built with Electron.</p>
```

Comment:
I understand that this is the description from the github readme but it's really bad appstore copy ;-) it'd be much more helpful for users to use the relevant copy from here for example: https://play.google.com/store/apps/details?id=org.jitsi.meet&hl=en_GB

Reply:
That's a really good idea, I didn't want to write something myself, but never thought to steal a description from somewhere else. (Disregarding the fact that the play store app is not exactly the same thing)